docs/ibm-granite.md

```diff
@@ -0,0 +1,87 @@
+
+## 🛠️ Getting Started
+
+Before getting started, ensure you have access to an IBM Granite model and have followed any other requirements in the [prerequisites](#prerequisites) above.
+
+The [ibm_granite_bee](/examples/agents/ibm_granite_bee.ts) example agent is set up to demonstrate how to use IBM Granite with the Bee Agent Framework. In order to run this example, take the following steps:
+
+> [!TIP]
+> More detailed installation instructions are available for step 1 in the [Location Installation](https://github.com/i-am-bee/bee-agent-framework?tab=readme-ov-file#local-installation) section of our main README
+
+1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the Bee Agent Framework git repository by running the following command:
+
+   ```shell
+   git clone https://github.com/i-am-bee/bee-agent-framework.git
+   yarn install --immutable
+   ```
+
+1. Copy the [.env.template](/.env.template) file to a new file called `.env`
+
+1. Fill out the required details within the `.env` file, as follows
+
+   1. For watsonx (replace the values shown in braces):
+
+      ```ini
+      LLM_BACKEND=watsonx
+      WATSONX_API_KEY={YOUR_WATSONX_API_KEY}
+      WATSONX_PROJECT_ID={YOUR_WATSONX_PROJECT_ID}
+      WATSONX_REGION=us-south
+      ```
+
+      Note: [other regions](https://www.ibm.com/docs/en/watsonx/saas?topic=integrations-regional-availability-cloud) are also available
+
+   1. For Ollama (replace the values shown in braces)
+
+      ```ini
+      LLM_BACKEND=ollama
+      OLLAMA_HOST={http://0.0.0.0:11434}
+      ```
+
+1. Run the [ibm_granite_bee](/examples/agents/ibm_granite_bee.ts) example:
+
+   ```shell
+   npm run start examples/agents/ibm_granite_bee.ts
+   ```
+
+   This will show the various stages of the agent running and ultimately deliver an answer similar to the following:
+
+   > Agent 🤖 : The current weather in London is 13.3°C with no rain. The maximum temperature for today is expected to be 13.5°C.
+
+   The default prompt is the question `What is the current weather in London?`
+
+   You can provide other prompts as follows:
+
+   ```shell
+    npm run start examples/agents/ibm_granite_bee.ts <<< 'Who is the president of the USA?'
+    npm run start examples/agents/ibm_granite_bee.ts <<< 'What is the spanish word for dog?'
+    npm run start examples/agents/ibm_granite_bee.ts <<< 'What does the alias command do in the BASH shell?'
+    npm run start examples/agents/ibm_granite_bee.ts <<< "What’s the largest technology company by market capitalization right now?"
+    npm run start examples/agents/ibm_granite_bee.ts <<< "What’s the weather going to be like tomorrow in Sydney?"
```

Comment:
We are using `yarn` in this project. Change to `yarn start examples/...`

Reply:
Good spot!  Resolved in [e808435](https://github.com/i-am-bee/bee-agent-framework/pull/229/commits/e8084350cf0daa37ddf5423d2f65cd0d2b58c887)